src/views/Bond/BondRedeem.jsx

```diff
@@ -0,0 +1,111 @@
+import { useSelector, useDispatch } from "react-redux";
```

Comment:
@dwjanus when I visit the Frax bond page, the Redeem V1 is not clickable. I don't think we want to show it, right?

![image](https://user-images.githubusercontent.com/81924824/125698198-2e5e3d3f-511b-46b3-b4cc-671888559e1c.png)

Reply:
i think youre right. I wasnt positive on if we were removing it so I set it to disabled while I got clarification.